### ISSUE TITLE
Quotemark rule incorrectly linting React properties

### ISSUE THREAD

**Issue by jameslee523** [palantir/tslint#2603]
### Bug Report

- __TSLint version__: 5.0.0+
- __TypeScript version__: 2.2.2
- __Running TSLint via__: CLI / VSCode

#### TypeScript code being linted

```ts
<Image resizeMode='contain' />
```

with `tslint.json` configuration:

```json
"quotemark": [true, "single", "avoid-escape"]
```

#### Actual behavior
Running the command `tslint -c tslint.json --project tsconfig.json` will produce the error: `' should be "`. VSCode also picks up these as errors.

This started occurring after updating from 4.5.1 to 5+. It looks to be an issue with React properties that are assigned to single quote values. Other quotes are correctly being linted such as `import React from 'react';`

#### Expected behavior
With 4.5.1 there was no error thrown here with the same config.

**Comment by ajafff** [comment]
@jameslee523 this will be fixed by #2605 
in the meantime you can add the option `"jsx-single"` to make it work again

**Comment by jameslee523** [comment]
Thanks for the quick response and solution. Adding the option worked for me.
